### PR TITLE
22 61 create profile component

### PR DESCRIPTION
**Pull request by nbarton76**
Edit user profile update

### CODE REVIEW

File: angular-ui/src/app/user-profile/user-profile.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<div>
+  <h1>{{ user.name }}</h1>
+  <p>{{ user.email }}</p>
+  <button (click)="editProfile()">Edit Profile</button>
```
Comment (nbarton76) [inline angular-ui/src/app/user-profile/user-profile.component.html:5]: I actually think a link would be better. If so : <a (click)="editProfile()" routerLink="/edit-profile">Edit Profile</a>

**Review by MindzEyeLove — APPROVED** [review]

File: angular-ui/src/app/app.module.ts [hunk 2]
```diff
@@ -26,13 +27,16 @@ import { DatePipe } from '@angular/common';
     MovieListComponent,
     MovieItemComponent,
     BrowseOptionComponent,
-
-    TimerComponent
+    UserProfileComponent
+  ],
+  imports: [
```
Comment (e-karr) [inline angular-ui/src/app/app.module.ts:32]: I don't know which merge this happened in, but the TimerComponent, FooterComponent, and WatchHistoryComponent need to be in the declarations array above. The BrowseOptionComponent is already there. Then this is a duplicate imports array that can be deleted.

File: angular-ui/src/app/app.module.ts [hunk 3]
```diff
@@ -26,13 +27,16 @@ import { DatePipe } from '@angular/common';
     MovieListComponent,
```
Comment (e-karr) [inline angular-ui/src/app/app.module.ts:27]: PasswordFormComponent needs to be in this declarations array

**Review by e-karr — CHANGES_REQUESTED** [review]
I left a few comments on fixes before I can approve this branch. I don't know if the errors are from a previous merge or not, but they prevent your branch from running

File: angular-ui/src/app/app.module.ts (hunk 3 above)
Comment (nbarton76) [reply angular-ui/src/app/app.module.ts:27]: Updated array

File: angular-ui/src/app/app.module.ts (hunk 2 above)
Comment (nbarton76) [reply angular-ui/src/app/app.module.ts:36]: Updated array

**Review by nbarton76 — COMMENTED** [review]
Ready for review

**Review by e-karr — CHANGES_REQUESTED** [review]
The only issues I'm seeing now is that PasswordFormComponent needs to be in the import list at the top of app.module.ts. Then lines 36-41 (the first imports array) can be deleted because it's a duplicate.

I'm sorry, I'm not sure how these errors happened since I know we don't do a lot of coding in app.module.ts and Angular adds things there for us automatically.

**Review by e-karr — APPROVED** [review]

**Pull request merged**